### PR TITLE
Update for charmhub, remove unused metadata keys

### PR DESCRIPTION
**Pull request by ca-scribner**

### CODE REVIEW

**Review by DnPlas — APPROVED** [review]
LGTM, let's resolve the CI issues.

File: charms/mlflow-server/metadata.yaml [hunk 1]
```diff
@@ -5,8 +5,6 @@ description: |
   MLflow
 summary: |
   A charm which provides a Kubernetes installation of the MLflow (https://mlflow.org/) server.
-series: [kubernetes]
-tags: [mlflow]
 resources:
   oci-image:
     type: oci-image
```
Comment (DnPlas) [inline charms/mlflow-server/metadata.yaml:10]: Seems like juju bundle deploy does not like the `auto-fetch` field below.

File: charms/mlflow-server/metadata.yaml [hunk 2]
```diff
@@ -5,13 +5,10 @@ description: |
   MLflow
 summary: |
   A charm which provides a Kubernetes installation of the MLflow (https://mlflow.org/) server.
-series: [kubernetes]
```
Comment (DnPlas) [inline charms/mlflow-server/metadata.yaml:8]: Should we leave `series` as per our discussion with @knkski ?

Comment (ca-scribner) [reply charms/mlflow-server/metadata.yaml:8]: agreed.  updating

File: bundle.yaml [hunk 3]
```diff
@@ -1,19 +1,22 @@
----
+bundle: kubernetes
+name: mlflow
+description: "MLflow"
+
 applications:
   mlflow-server:
-    charm: "cs:mlflow-server"
+    charm: "ch:mlflow-server"
+    channel: "latest/edge"
```
Comment (DnPlas) [inline bundle.yaml:8]: Should we add [this](https://github.com/canonical/argo-operators/pull/17/files#diff-73c670418fb192fb77d3a1c95ada8448ee256f1a6afa0e6ede380a7c63221184R15)?

**Review by DnPlas — COMMENTED** [review]
Just one comment

**Pull request merged**